Sources/Amplitude/Amplitude.m

```diff
@@ -753,11 +753,8 @@ - (void)uploadEventsWithLimit:(int)limit {
         
         long identifyCount = [self->_identifyBuffer count];
         long numIdentify = limit > 0 ? fminl(identifyCount, limit) : identifyCount;
-        
         NSMutableArray *events = [[self->_eventsBuffer subarrayWithRange:NSMakeRange(0, numEvents)] mutableCopy];
         NSMutableArray *identifys = [[self->_identifyBuffer subarrayWithRange:NSMakeRange(0, numIdentify)] mutableCopy];
-        NSDictionary *merged = [self mergeEventsAndIdentifys:events identifys:identifys numEvents:(numEvents+numIdentify)];
```

Comment:
Before We store the events into file storage after we merged the events.
But the mergeEventsAndIdentifys will remove all events in the events array. We should move the merge event after the store Event and store Identify.

Reply:
good catch, this should be caught earlier since this seems breaking the storage